### PR TITLE
Leader should poll follower for its channel.

### PR DESCRIPTION
**Pull request by wingo**

### CODE REVIEW

**Review by dpino — APPROVED** [review]

File: src/apps/config/leader.lua [hunk 1]
```diff
@@ -683,9 +683,8 @@ function Leader:send_messages_to_followers()
    for _,follower in ipairs(self.followers) do
       if not follower.channel then
          local name = '/'..tostring(follower.pid)..'/config-follower-channel'
-         -- local success, channel = pcall(channel.open, name)
-         --if success then follower.channel = channel end
-         follower.channel = channel.open(name)
+         local success, channel = pcall(channel.open, name)
+         if success then follower.channel = channel end
```
Comment (teknico) [inline src/apps/config/leader.lua:687]: A little delay in the `else` of this `if` would avoid a probably too quick polling loop.

Comment (dpino) [reply src/apps/config/leader.lua:687]: The delay exists indirectly via Leader:pull(). `send_message_to_followers` is polled every `period` seconds.

Comment (teknico) [reply src/apps/config/leader.lua:687]: Ah, right. And it's one loop for all followers, it wouldn't be a good idea anyway.

**Comment by teknico** [comment]
I need this, merging it myself.

**Pull request merged**